backend/s390x.pm

```diff
@@ -18,6 +18,8 @@ use backend::VNC;
 
 use testapi qw(get_var check_var set_var);
 
+use Net::Address::IP::Local;
```

Comment:
not properly packaged anyways. Note that this module tries to connect to a server in the internet to find out it's own address.